docs/Contributing/API-for-contributors.md

```diff
@@ -4546,3 +4550,73 @@ param, this endpoint is considered a documented REST API endpoint
   "settings": {"hidden_host_columns": ["hostname"]},
 }
 ```
+
+## Conditional access
+
+### Initiate Microsoft Entra conditional access
+
+Kick off authentication with Microsoft Entra to configure conditional access.
+
+`POST /api/v1/conditional-access/microsoft`
+
+#### Parameters
+
+| Name       | Type   | In   | Description                                                                                                                 |
+| ---------- | ------ | ---- | --------------------------------------------------------------------------------------------------------------------------- |
+| microsoft_tenant_id      | string | body | **Required.** The Microsoft Entra tenant ID. |
+
+
+
+##### Request body
+
+```json
+{
+  "fleet_license_key": "<LICENSE KEY>",
+  "microsoft_tenant_id": "<TENANT ID>"
+}
+```
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+  "microsoft_authentication_url": "https://login.microsoftonline.com/<TENANT ID>/adminconsent?client_id=<CLIENT ID>"
+}
+
+```
+
+
+### Confirm Microsoft Entra conditional access configuration
+
+`POST /api/v1/conditional-access/confirm`
```

Comment:
Double checking: Should this be a `GET`? (it's not changing anything server side, it's just getting status)

Reply:
@jacobshandling @rachaelshaw

Reply:
Or is this the second post to finalize the setup (after admin consented)?

Reply:
I think having the admin consent is the final step in setup, `GET` makes sense to me here too